redwood-layout-shared-test/src/commonMain/kotlin/app/cash/redwood/layout/AbstractBoxTest.kt

```diff
@@ -86,6 +86,13 @@ abstract class AbstractBoxTest<T : Any> {
     verticalAlignment = CrossAxisAlignment.Start,
   )
 
+  @Test
+  fun testWrapWithChildren_Wrap_Stretch_Start() = testWithChildren(
```

Comment:
I noticed that all these tests specify `Wrap` twice in the name. Below we use `Fill` but then `Wrap`.

Will rename in a follow-up.

Reply:
https://github.com/cashapp/redwood/pull/1844